README.md

```diff
@@ -58,6 +61,12 @@ To install, clone down this repo and run `npm link`. This will bind `apollos` to
   - `--production`: Run the application in production mode
   - `--debug`: Run the application in debug mode
 
```

Comment:
@JakeDawkins I don't believe `yarn link` works. So keeping it as `npm link` is great. 

[npm link](https://docs.npmjs.com/cli/link)